r/DESCRIPTION

```diff
@@ -62,6 +62,7 @@ Suggests:
     lubridate,
     pillar,
     pkgload,
+    readr,
```

Comment:
Not sure the test is worth adding to suggests, though we will get rid of that annoying xref error if we do.